src/components/LocationList.js

```diff
@@ -0,0 +1,13 @@
+import { Location } from './Location';
+
+export const LocationList = ({ locations = [] }) => {
+  if (locations.length === 0) return <p>List is empty.</p>;
+
+  return (
+    <ul className="list__locations">
+      {locations?.map((location) => (
```

Comment:
I don't think you can use optional chaining here😄

Reply:
You won't need to use optional chaining because you've already used a default of an empty array above ^. You can use optional chaining in this way.

Reply:
Thank you, Katia and Lauren! We will remove it.